### PR TITLE
fixed some bugs on volume and number of volumes

### PR DESCRIPTION
**Pull request by smarchand**

### CODE REVIEW

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 1]
```diff
@@ -173,9 +173,9 @@
             <date-part name="year"/>
           </date>
           <group>
-            <text term="volume" form="short" suffix="."/>
-            <text variable="volume" prefix=". " suffix="/"/>
-            <text variable="number-of-volumes"/>
+              <text term="volume" form="short" suffix=".&#160;"/>
+            <text variable="volume" />
```
Comment (adam3smith) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: please fix the spacing here (you added an extra space at the beginning of the line.

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 2]
```diff
@@ -173,9 +173,9 @@
...
+              <text variable="number-of-volumes" prefix="/"/>
```
Comment (adam3smith) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: ditto

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 3]
```diff
@@ -190,9 +190,9 @@
             <date-part name="year" suffix=", "/>
           </date>
           <group>
-            <text term="volume" form="short" suffix=". "/>
-            <text variable="number-of-volumes" prefix=". " suffix="/"/>
-            <text variable="volume" suffix=","/>
+              <text term="volume" form="short" suffix=".&#160;"/>
```
Comment (adam3smith) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: ditto

File: ens-de-lyon-centre-d-ingenierie-documentaire.csl [hunk 4]
```diff
@@ -190,9 +190,9 @@
...
+            <text variable="volume"  suffix="/"/>
```
Comment (adam3smith) [inline ens-de-lyon-centre-d-ingenierie-documentaire.csl]: this seems like you'd want to do it with a delimiter, i.e.

```
<group delimiter="/" suffix=", "/>
  <text variable="volume"/>
  <text variable="number-of-volumes/>
</group>
```

Also, same issue with extra space.

**Comment by adam3smith** [comment]
you can update your pull request using the pencil button here:
https://github.com/smarchand/styles/tree/patch-12/ens-de-lyon-centre-d-ingenierie-documentaire.csl 
Let us know when you're done.

**Comment by smarchand** [comment]
OK I made the changes for the extra space. Concerning the delimiter, I didn't manage to make this change, so I let the style unchanged since it was working. I will test later your suggestion with more time.
Can you merge the style now ?
all the best
S.

**Comment by rmzelle** [comment]
As you can see in https://github.com/citation-style-language/styles/pull/1658/files, your indentation is not consistent with the rest of the style. We don't use tabs to indent, just spaces (two per indentation level). You should be able to use http://rintze.zelle.me/style-formatter/ to quickly reindent your style. Maybe you could try that webpage and update your pull request with a properly indented version?

**Comment by smarchand** [comment]
ok ; I used oxygen which apparently made a mess with that.
I will do it
thanks !

2015-09-18 11:31 GMT+02:00 Rintze M. Zelle <notifications@github.com>:

> As you can see in
> https://github.com/citation-style-language/styles/pull/1658/files, your
> indentation is not consistent with the rest of the style. We don't use tabs
> to indent, just spaces (two per indentation level). You should be able to
> use http://rintze.zelle.me/style-formatter/ to quickly reindent your
> style. Maybe you could try that webpage and update your pull request with a
> properly indented version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1658#issuecomment-141398326>
> .
>



-- 
Stéphane

**Comment by rmzelle** [comment]
If you are using an XML editor like oXygen it should also be pretty straightforward to change the indentation from tabs to spaces in the editor, by the way.

**Comment by rmzelle** [comment]
Thanks!

For future edits, it would be good to use delimiters as much as possible, as @adam3smith already mentioned. E.g. you can rewrite

```xml
          <group>
            <text term="volume" form="short" suffix=".&#160;"/>
            <text variable="volume"/>
            <text variable="number-of-volumes" prefix="/"/>
          </group>
```

as

```xml
          <group delimiter="/">
            <group delimiter=".&#160;">
              <text term="volume" form="short"/>
              <text variable="volume"/>
            </group>
            <text variable="number-of-volumes"/>
          </group>
```

With the latter, you get correct punctuation even if one of the variables has no value.

**Pull request merged**